packages/wonder-blocks-layout/components/media-layout.js

```diff
@@ -168,7 +168,9 @@ class MediaLayoutInternal extends React.Component<CombinedProps, State> {
 
             // And then through each key of each stylesheet
             for (const name of Object.keys(styleSheet)) {
-                if (mockStyleSheet.hasOwnProperty(name)) {
+                if (
+                    Object.prototype.hasOwnProperty.call(mockStyleSheet, name)
```

Comment:
This was the only new violation.

https://eslint.org/docs/rules/no-prototype-builtins